R/read-dese-childhood-services.R

```diff
@@ -18,14 +21,14 @@ read_dese_childhood_services <- function(url = urls$read_dese_childhood_services
 
   search_term <- 'child-care-australia-report'
 
-  links <- djprdata:::get_latest_download_url(url, search_term) %>%
```

Comment:
good pick-up here, it is bad practice to use `:::` to access internal functions of the package you are working on.